### PR TITLE
US 1583733 Add missing language IDs - 54

### PR DESCRIPTION
**Pull request by v-thpra**
US 1583733 - Fix CATS P1 issues by adding missing language IDs to code blocks.
- Used "console" for sample program output displayed on the console.
- Used "csv" for example of csv input to a sample program.
- Used "xml" for sample program output that is mostly XML with some text.

Contributes to #2192

### CODE REVIEW

File: docs/visual-basic/programming-guide/concepts/linq/how-to-project-an-anonymous-type.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ In some cases you might want to project a query to a new type, even though you k
   
  Anonymous types are the C# implementation of the mathematical concept of a *tuple*. The mathematical term tuple originated from the sequence single, double, triple, quadruple, quintuple, n-tuple. It refers to a finite sequence of objects, each of a specific type. Sometimes this is called a list of name/value pairs. For example, the contents of an address in the [Sample XML File: Typical Purchase Order (LINQ to XML)](../../../../visual-basic/programming-guide/concepts/linq/sample-xml-file-typical-purchase-order-linq-to-xml.md) XML document could be expressed as follows:  
   
-```  
+```xml
```
Comment (Youssef1313) [inline docs/visual-basic/programming-guide/concepts/linq/how-to-project-an-anonymous-type.md]: I don't think xml is correct here.

File: docs/visual-basic/programming-guide/concepts/linq/how-to-retrieve-the-value-of-an-attribute-linq-to-xml.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ End Module
   
  This example produces the following output:  
   
-```  
+```xml
```
Comment (Youssef1313) [inline docs/visual-basic/programming-guide/concepts/linq/how-to-retrieve-the-value-of-an-attribute-linq-to-xml.md]: I don't think xml is correct here too.

Comment (mairaw) [reply docs/visual-basic/programming-guide/concepts/linq/how-to-retrieve-the-value-of-an-attribute-linq-to-xml.md]:
```suggestion
```console  
```

**Review by mairaw — APPROVED** [review]
Thanks @v-thpra. Left a few suggestions.

**Review by mairaw — CHANGES_REQUESTED** [review]
Oops, clicked the wrong button...

**Comment by v-thpra** [comment]
@mairaw Thanks for the review! I implemented your changes.

**Review by mairaw — APPROVED** [review]
Thank you @v-thpra!

**Pull request merged**